src/lightning/pytorch/loops/__init__.py

```diff
@@ -11,9 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from lightning.pytorch.loops.loop import _Loop  # noqa: F401 isort: skip (avoids circular imports)
```

Comment:
Moving this is what is causing the circular import issues. I assume one of the tools did it accidentally? isort should be skipping it

Reply:
I ran with isort version 5.10.1. Maybe that was the issue.